omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
As I said earlier, you need to put the uuid in quotes and you need to look at the generated JS, of course the logic that evaluates the `'{{patient.uuid}}'` expression is going to try to escape the quotes, use html character entities in place of the quoetes to avoid this, so you need to the write the expression as `&#x27;{{patient.uuid}}&#x27;` instead of `'{{patient.uuid}}'`